snap/snapcraft.yaml

```diff
@@ -153,7 +153,8 @@ parts:
 
   freecad:
     plugin: cmake
-    source: https://github.com/FreeCAD/FreeCAD.git
+    source: https://github.com/luzpaz/FreeCAD.git
+    source-branch: PR18599
```

Comment:
This doesn't look right, I can't find that branch in your fork

Reply:
I believe this should be

```suggestion
    source: https://github.com/ein-shved/FreeCAD.git
    source-branch: about-thumbnail
```

Reply:
:facepalm: